src/content-routing/index.js

```diff
@@ -35,34 +41,28 @@ class ContentRouting {
   }
 
   /**
-   * Iterates over all content routers in series to find providers of the given key.
-   * Once a content router succeeds, iteration will stop.
+   * Iterates over all content routers in parallel to find providers of the given key.
```

Comment:
Can you change the above comment in the constructor that makes the DHT first? As now the order is irrelevant

Reply:
Done!